src/test/java/com/github/beatngu13/visualtesting/util/ElementCount.java

```diff
@@ -0,0 +1,16 @@
+package com.github.beatngu13.visualtesting.util;
+
+import org.openqa.selenium.WebDriver;
+
+import de.retest.web.selenium.By;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ElementCount {
```

Comment:
As we use this class for possibly more stats, I suggest renaming it to `Statistics`.